web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -233,6 +233,7 @@ public class QueryExecutorBean implements QueryExecutor {
     private SecurityMarking marking;
     
     @Inject
+    @SpringBean(name = "ResponseObjectFactory")
```

Comment:
Is this `@SpringBean` required here?

Reply:
yes we need a spring bean here. This is an EJB that is created via CDI, and the response object factory is a spring created object.